### ISSUE TITLE
Exceeding upload limit returns HTTP 500 response with no details in log

### ISSUE THREAD

**Issue by berni2288** [gentics/mesh#505]
## Gentics Mesh Version, operating system, or hardware.
- v0.22.2

## Operating System
- Linux

## JVM
- JDK from Mesh container

## Problem

HTTP 500 response is returned when a big file is uploaded that exceeds the upload byteLimit setting
The log does not contain any detailled error message except one ERROR line.

Mesh UI says "Interner Fehler aufgetreten"

```
mesh_1           | 12:49:09.482 [Velvety Chimecho] ERROR [vert.x-eventloop-thread-4] - Error for request in path: /api/v1/portalphp/nodes/4aca1a2bbb2311e898eb0242ac1b0003/binary/binarycontent?lang=en
mesh_1           | 12:49:09.483 [Velvety Chimecho] ERROR [vert.x-eventloop-thread-4] - 172.18.0.1 - - [Wed, 26 Sep 2018 12:49:09 GMT] "POST /api/v1/portalphp/nodes/4aca1a2bbb2311e898eb0242ac1b0003/binary/binarycontent?lang=en HTTP/1.1" 500 48 "http://localhost:8081/mesh-ui/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36"
```

Increasing the upload limit to above the filesize works.

## Reproducer

Upload a big file in Mesh UI that exceeds the configured upload limit (upload.byteLimit).
After clicking on "Save & publish" an error is returned.

## Expected behaviour and actual behaviour

HTTP 413 - Request Entity Too Large should be returned

**Comment by Jotschi** [comment]
It seems the 413 is not handled correctly:
https://github.com/vert-x3/vertx-web/blob/master/vertx-web/src/main/java/io/vertx/ext/web/handler/impl/BodyHandlerImpl.java#L171

**Comment by Jotschi** [comment]
Will be fixed with 0.24.1 release.
